nightfall-optimist/src/services/verify.mjs

```diff
@@ -0,0 +1,85 @@
+import childProcess from 'child_process';
+import jsonfile from 'jsonfile';
+import fs from 'fs';
+
+const { spawn } = childProcess;
+const { writeFile } = jsonfile;
+
+const deleteSingleFile = fileName => {
+  fs.unlink(fileName, err => {
+    if (err) throw err;
+  });
+};
+
+/**
+ * Takes in a proof and a verification key and determines if the proof verifies.
+ *
+ * @param {String} provingScheme - Available options are 'g16', 'pghr13', 'gm17'
+ * @param {String} backEnd - Available options are 'libsnark', 'bellman', 'ark'
+ * @param {Object} [options] - Options for output
+ * @param {Boolean} options.createFile - Whether or not to output a json file
+ * @param {String} [options.directory=./] - Directory to output files in
+ * @param {String} [options.fileName=proof.json] - Name of JSON proof file ()
+ */
+export default async function verify({
+  vk,
+  proof,
+  inputs,
+  provingScheme = 'gm17',
```

Comment:
Switch to G16 and bellman to align with changes in #326 ?

Reply:
Done, thanks for the reminder!